apps/voting/app/src/components/SummaryRows.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react'
```

Comment:
Ah, sorry for the rework—we are likely to have a conflict here against https://github.com/aragon/aragon-apps/pull/1113.

We'll merge that one first, and then it should be pretty easy to rollback these changes against Voting here to solve any conflicts :).